pylearn2/datasets/dataset.py

```diff
@@ -18,7 +18,7 @@ def __iter__(self):
         return self.iterator()
 
```

Comment:
I don't think we should be required to take files off the whitelist every time we update the documentation, but sure, I'll try to find time for that.